default/python/AI/colonization/nest_rating.py

```diff
@@ -9,5 +9,5 @@ def special_is_nest(special_name: str) -> bool:
     return special_name in _nest_ratings
 
 
-def get_nest_rating(special_name: str, default: float) -> float:
-    return _nest_ratings.get(special_name, default)
+def get_nest_rating(special_name: str) -> float:
+    return _nest_ratings.get(special_name, 0)
```

Comment:
This looks a bit more consistent, with return annotation
```suggestion
    return _nest_ratings.get(special_name, 0.0)
```

Reply:
Well, then let's make the actual ratings floats as well.